### DISCUSSION TITLE
TwilightDelight-forge-1.18.2-1.2.0-Beta

### DISCUSSION THREAD

**Discussion by Goulixiaoji** [Laokuai233/Twilight-Delight#14] [Ideas]
<hr /><em>This discussion was created from the release <a href='https://github.com/Laokuai233/Twilight-Delight/releases/tag/v1.2.0-1.18.2-forge-beta'>TwilightDelight-forge-1.18.2-1.2.0-Beta</a>.</em>

**Comment by moechakucha** [comment]
Tip: You can actually avoid opening a discussion when creating a release

**Reply by Goulixiaoji** [reply]
Yes, I know. Thank you.(But why we chat in English?)
